geotrellis-tiling/src/main/scala/geotrellis/geotrellis-tiling/package.scala

```diff
@@ -0,0 +1,36 @@
+package geotrellis
+
+import geotrellis.raster._
+import geotrellis.vector.Extent
+import geotrellis.proj4._
+import geotrellis.util._
+
+import org.locationtech.proj4j.UnsupportedParameterException
+
+package object tiling extends Implicits {
+  type TileBounds = GridBounds[Int]
+
+  type SpatialComponent[K] = Component[K, SpatialKey]
+  type TemporalComponent[K] = Component[K, TemporalKey]
+
+  private final val WORLD_WSG84 = Extent(-180, -89.99999, 179.99999, 89.99999)
+
+  implicit class CRSWorldExtent(crs: CRS) {
```

Comment:
`extends AnyVal`?

Reply:
Would that be beneficial? We don't have that in our current code.

Reply:
Yes https://docs.scala-lang.org/overviews/core/value-classes.html 
It won't be bad to add it, but not it's an obligatory thing in terms of this PR for sure.